### ISSUE TITLE
AbstractMethodError IDEA 15 EAP Ultimate Mac OSX Yosemite

### ISSUE THREAD

**Issue by mmoayyed** [andrepdo/findbugs-idea#104]
See this every time IDE loads:

```
org.twodividedbyzero.idea.findbugs.core.FindBugsCompileAfterHook$1.beforeBuildProcessStarted(Lcom/intellij/openapi/project/Project;Ljava/util/UUID;)V
java.lang.AbstractMethodError: org.twodividedbyzero.idea.findbugs.core.FindBugsCompileAfterHook$1.beforeBuildProcessStarted(Lcom/intellij/openapi/project/Project;Ljava/util/UUID;)V
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.util.messages.impl.MessageBusConnectionImpl.deliverMessage(MessageBusConnectionImpl.java:116)
	at com.intellij.util.messages.impl.MessageBusImpl.doPumpMessages(MessageBusImpl.java:327)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpMessages(MessageBusImpl.java:314)
	at com.intellij.util.messages.impl.MessageBusImpl.sendMessage(MessageBusImpl.java:293)
	at com.intellij.util.messages.impl.MessageBusImpl.access$200(MessageBusImpl.java:41)
	at com.intellij.util.messages.impl.MessageBusImpl$2.invoke(MessageBusImpl.java:182)
	at com.sun.proxy.$Proxy125.beforeBuildProcessStarted(Unknown Source)
	at com.intellij.compiler.server.BuildManager.a(BuildManager.java:1175)
	at com.intellij.compiler.server.BuildManager.access$2800(BuildManager.java:131)
	at com.intellij.compiler.server.BuildManager$14.call(BuildManager.java:931)
	at com.intellij.compiler.server.BuildManager$14.call(BuildManager.java:923)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:37)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
	at org.jetbrains.ide.PooledThreadExecutor$1$1.run(PooledThreadExecutor.java:55)
```

**Comment by Donnerbart** [comment]
I get this error every time I run a test.

IDEA 15 EAP
Ubuntu Ubuntu 14.04.2 LTS (64-Bit)

**Comment by retomerz** [comment]
Will be fixed in 0.9.997

**Comment by retomerz** [comment]
Fixed in version 0.9.997

**Comment by mmoayyed** [comment]
Thanks. Confirmed fix.
